changelog/fragments/confg-3-alpha-removed.yaml

```diff
@@ -0,0 +1,30 @@
+entries:
+  - description: >
+      PROJECT config version 3-alpha has been upgraded to version 3
+    kind: change
+    breaking: true
+    migration:
+      header: PROJECT config version 3-alpha must be upgraded to 3
+      body: >
+        Version 3-alpha has been removed after version 3 was released. While this change
+        is not technically breaking because the spec at that version was alpha, it was
+        used by default in `operator-sdk` commands so should be marked as breaking
+        and have a convenient migration path. The `config3alphato3` command will convert
+        most of your PROJECT file from version 3-alpha to 3, and leave comments with
+        directions where automatic conversion is not possible:
+
+        ```console
+        $ cat PROJECT
+        version: 3-alpha
+        resources:
+        - crdVersion: v1
+        ...
+        $ operator-sdk config3alphato3
```

Comment:
We have an EP to create a command to migrate a pkgmanifest to a bundle
We might have many scenarios that we want to migrate something. So, imo we need an `operator-sdk migrate`.

Also, I think we need to link the migration guide in the upstream docs as well.

Reply:
See : https://github.com/operator-framework/enhancements/pull/64/files

c/c @jmrodri